app/src/controllers/TorusController.js

```diff
@@ -181,6 +182,13 @@ export default class TorusController extends EventEmitter {
         opts.rehydrate()
       }, 50)
     }
+
+    // // channel method management
+    this.channelController = new ChannelController({
+      keyringController: this.keyringController,
+      networkController: this.networkController,
+      store: this.store
```

Comment:
Please don't pass in main store.
create a new observable store in the ChannelController and use it for internal purposes.
Then, hook up this store into the TorusController.
See other controllers for reference.